packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -98,3 +196,33 @@ blockquote {
   border-right: 1px solid cv("gray", "300");
   border-radius: 4px;
 }
+
+textarea::-webkit-input-placeholder,
+input::-webkit-input-placeholder {
+  font-family: "Aeonik";
```

Comment:
Curious why these are needed explicitly, since I would assume the `textarea` and `input` elements should also have the same font, and the placeholder pseudoclasses would inherit that font value.

Reply:
removed refinement for placeholder, change to global input + textarea